micro_dl/preprocessing/readme.md

```diff
@@ -44,6 +50,12 @@ The following settings can be adjusted in preprocessing using a config file (see
 The tiling class will take the 2D image files, assemble them to stacks in case 3D tiles are required,
 and store them as tiles based on input tile size, step size, and depth.
 
+#### Specifications of metadata to be generated
+
+* metadata
+  * order (str): If 'cztp', the images produced by processing are named by the format 'channel_zslice_time_position', for example, 'c001_z046_t000_p023'
+  * name_parser (str): 'parse_sms_name' corresponds to 'sms' image naming format 'img_channelname_t***_p***_z***_extrafield', default naming convention is 'im_c***_z***_t***_p***'
+
 All data will be stored in the specified output dir, where a 'preprocessing_info.json' file
```

Comment:
The sentence is not complete. Perhaps define here what the json file is storing.